scripts/install

```diff
@@ -29,11 +31,9 @@ function check_strimzi {
 function deps {
     local -r added_p=$(helm repo list | grep -s google)
```

Comment:
looks like we can remove this local variable now that we're _definitely_ installing the google remote chart repo for ingress et al.

Reply:
good call